### DISCUSSION TITLE
Allow conditional to check for a lack of an item

### DISCUSSION THREAD

**Discussion by Foxtrot39** [FakeFishGames/Barotrauma#10841] [Ideas and suggestions]
Currently we got requireditem to check for specific items to be present but there is no way to check for the lack of certain items

Lets say I want a light to turn on when item X is contained alongside other parameters and turn off when absent with other conditionals

The conditional to check for the lack of an item is just getting entirely ignored by the game

**Comment by ubertpendragon** [comment]
Talents can invert their conditional, I'm not sure about statuseffect conditionals

`        <AbilityConditionHasItem tags="clothing,deepdiving" invert="true" />`
